Resources/config/provider-phpcr.xml

```diff
@@ -22,6 +22,8 @@
             <call method="setRouteCollectionLimit"><argument>%cmf_routing.route_collection_limit%</argument></call>
         </service>
 
+        <service id="cmf_routing.route_provider" alias="cmf_routing.phpcr_route_provider"/>
```

Comment:
They were first done in the extension, but it makes no sense to do it in the extension (as they are just static).